benchmarking/utils/build_program.py

```diff
@@ -30,7 +30,8 @@ def buildProgramPlatform(dst, repo_dir, framework, frameworks_dir, platform):
         os.chmod(dst, 0o777)
     print(result)
 
-    if not os.path.isfile(dst):
+    if not os.path.isfile(dst) and \
+            (not (os.path.isdir(dst) and platform[0:4] == "ios")):
```

Comment:
0:2? maybe you can use startswith

Reply:
0:4, the first 4 letters. :4 is also fine.